docs/dev/compile/Compile.rst

```diff
@@ -89,7 +89,7 @@ assistance.
 All Platforms
 =============
 Before you can compile the code you'll need to configure your build with cmake. Some IDEs can do this,
-but from command line is the usual way to do this; thought the Windows section below points out some
+but from command line is the usual way to do this; though the Windows section below points out some
```

Comment:
while this is correct, the sentence is still pretty cringeworthy.  it would be better rewritten as two sentences rather than this awkward semicolon splice

Reply:
Fair point, I've gone ahead and re-written the sentence.